django3_codemods/config_tools/tox_modernizer.py

```diff
@@ -92,8 +93,13 @@ def _replace_django_versions(self):
         self.config_parser[TEST_ENV_SECTION][TEST_ENV_DEPS] = dependencies
 
     def _update_config_file(self):
+        # ConfigParser insists on using tabs for output. We want spaces.
+        with io.StringIO() as configw:
+            self.config_parser.write(configw)
```

Comment:
is `configw` a typo?

Reply:
I meant "config file to Write to" but now that you mention it, it looks weird :)